.circleci/config.yml

```diff
@@ -0,0 +1,79 @@
+version: 2.0
+
+jobs:
+  build-deploy: # build for the master branch
+    machine: true
+    working_directory: ~/redash-ui-tests
+    steps:
+      - checkout
+      - run:
+          name: Build docker image and push to repo
+          command: |
+            docker version
+            docker build -t mozilla/redash-ui-tests:latest .
+            docker login -u $DOCKERHUB_USER -p $DOCKERHUB_PASS
+            docker push mozilla/redash-ui-tests:latest
+ 
+  build-release: # build for tags
+    machine: true
+    working_directory: ~/redash-ui-tests
+    steps:
+      - checkout
+      - run:
+          name: Build docker image and push to repo
+          command: |
+            docker version
+            docker build -t mozilla/redash-ui-tests:latest .
+            docker login -u $DOCKERHUB_USER -p $DOCKERHUB_PASS
+            docker push mozilla/redash-ui-tests:${CIRCLE_TAG}
+  integration_test:
+    working_directory: ~/redash-ui-tests
+    machine: true
+    steps:
+      - checkout
+      - run:
+           name: Clone upstream redash
+           command: git clone git@github.com:getredash/redash.git --depth 1
```

Comment:
Do we need to git clone the project here?

We use the stable release from DockerHub for the [server image](https://github.com/mozilla/redash-ui-tests/blob/master/docker-compose.yml#L10) in docker-compose.

Reply:
Ah yes, we could!